### ISSUE TITLE
Group Member Attributes no longer available to show in grid on Event Registration Forms

### ISSUE THREAD

**Issue by charlottedean** [SparkDevNetwork/Rock#4448]
### Description
When setting up event registration, a group member attribute used to be shown is available to 'Show in Grid'. Now once you select the group member attribute, that option goes away.

### Steps to Reproduce
1. Event Registrations
1. Create new template
1. Click Registrant Form, click add Registrant Form Field
1. Under source select Group Member Attribute and notice the 'Show in Grid' box disappears.

**Expected behavior:**

The 'Show in Grid' box should still display.

**Actual behavior:**

The 'Show in Grid' box disappears when Group Member Attribute is selected.

### Versions
* **Rock Version:** 11.1
* **Client Culture Setting:** en-US

**Comment by jonedmiston** [comment]
Confirmed still an issue on v16.0

**Comment by jonedmiston** [comment]
"Fixed" in v16.1, but this limitation was added as a fix to this issue (https://github.com/SparkDevNetwork/Rock/issues/3725). I reverted the limitation and everything does appear to work. There has been a lot of updates in this area so it's possible those updates addressed the original problem. 

If the original problem arises again we'll look at it. We're now much better about keeping tech notes 😉 .
